pilot/pkg/serviceregistry/kube/controller/controller.go

```diff
@@ -604,6 +601,8 @@ func (c *Controller) Run(stop <-chan struct{}) {
 	go c.imports.Run(stop)
 	go c.exports.Run(stop)
 
+	c.networkManager.watchGatewayResources(c, stop)
```

Comment:
donot we only need to watch gateway for config cluster?

Reply:
We need to watch all clusters to discover their multi-network gateways. 
If the other clusters don't have the CRD, this won't do anything.

Reply:
A little bit different from what istio does today, we do not support any other CR from remote clusters. So want to hear more voices about this

@howardjohn @ramaraochavali WDYT

Reply:
Yes, this is the only one now. Not quite sure if we should we expand it to gateway api.